### PR TITLE
Added `SellingPrice`

### PR DESCRIPTION
**Pull request by iaronaraujo**
#### What does this PR do? \*

Implementation of `SellingPrice`

#### How to test it? \*

[Workspace](https://productprice--storecomponents.myvtex.com/gorgeous-watch/p)
You can also edit this value on site-editor

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
```
Comment (kevinch) [inline CHANGELOG.md]: Are you using prettier? There should be a line break between those two titles.

File: messages/pt.json [hunk 2]
```diff
@@ -1,9 +1,12 @@
 {
   "store/list-price.default": "Preço original: {listPriceValue}",
-  "store/savings.default": "Preço de venda: {newPriceValue}",
+  "store/selling-price.default": "Preço de venda: {sellingPriceValue}",
+  "store/savings.default": "Economize {savingsValue} ({savingsPercentage})",
   "store/installments.default": "ou {installmentsNumber}x de {installmentValue}{hasInterest, select, true {, {interestRate} de juros} false { sem juros}}",
   "admin/list-price.description": "Valores disponiveis para interpolação: '{listPriceValue}'",
   "admin/list-price.title": "List Price",
+  "admin/selling-price.description": "Valores disponiveis para interpolação: '{sellingPriceValue}'",
+  "admin/selling-price.title": "Selling Price",
```
Comment (kevinch) [inline messages/pt.json]: English?

File: CHANGELOG.md (hunk 1 above)
Comment (iaronaraujo) [reply CHANGELOG.md]: We decided to keep the changelogs like this in our apps, but we can discuss this further.

File: messages/pt.json (hunk 2 above)
Comment (iaronaraujo) [reply messages/pt.json]: I am going to fix all these translation problems in the next PR, but thanks for commenting!

File: CHANGELOG.md (hunk 1 above)
Comment (kaisermann) [reply CHANGELOG.md]: @kevinch It's something @klzns showed at some FEG meeting a while ago. Sometimes that empty line can lead to a annoying conflict :l

File: react/SellingPrice.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  if (!selectedItem) {
+    return null
+  }
+
+  const commertialOffer = selectedItem?.sellers[0]?.commertialOffer
+  if (!commertialOffer) {
+    return null
+  }
+
+  const sellingPriceValue = commertialOffer.Price
```
Comment (kaisermann) [inline react/SellingPrice.tsx]:
```suggestion
  const commercialOffer = selectedItem?.sellers[0]?.commertialOffer
  if (!commercialOffer) {
    return null
  }

  const sellingPriceValue = commercialOffer.Price
```

**Pull request merged**